pkg/auth/iam/iam.go

```diff
@@ -477,3 +500,44 @@ func respondError(httpStatus, errorCode int, errorMessage string) restful.Servic
 		Message: string(messageByte),
 	}
 }
+
+// SetDevStackTracer used for activate verbose error message in non-prod environment
+func SetDevStackTracer() {
+	DevStackTraceable = true
+	realmName, realmNameExists := os.LookupEnv("REALM_NAME")
+	if !realmNameExists {
+		DevStackTraceable = false
+		logrus.Warn("unable to get REALM_NAME environment variable, DevStackTracer disabled")
+	}
+
+	realmLives, realmLiveExists := os.LookupEnv("REALM_LIVE")
+	if !realmLiveExists {
+		realmLives = constant.DisabledDevStackTracerEnv
+	}
+	realmLiveArr := strings.Split(realmLives, ",")
+
+	for _, realmLive := range realmLiveArr {
```

Comment:
strings.Contains for more readable code
https://www.geeksforgeeks.org/string-contains-function-in-golang-with-examples/

Reply:
yes, Just realize that it can be simplified 
Thanks for pointing that @rexydwybrundy